etl/command.py

```diff
@@ -180,6 +180,11 @@ def run_dag(
 
     steps = compile_steps(dag, includes, excludes, downstream=downstream, only=only)
 
+    if not steps:
```

Comment:
the other option: check this in the `filter_to_subgraph` method. there, we could verify that every step in `includes` actually matches something in the dag. that would allow us to throw a more specific error - ie in the case that someone passed multiple things to include, we could tell them which one specifically didn't match.

i think the simple way is enough for now though?

Reply:
Right, error makes more sense.

Reply:
Simple is better, if someone is running multiple steps then she's an advanced user and knows what to except.